packages/compass-assistant/src/compass-assistant-provider.spec.tsx

```diff
@@ -84,7 +84,8 @@ describe('AssistantProvider', function () {
     );
   });
 
-  describe('with existing chat instance', function () {
+  // TODO: some internal logic in lg-chat breaks all these tests, re-enable the tests
+  describe.skip('with existing chat instance', function () {
```

Comment:
@gagik @lerouxb  just wanted to highlight: I can either revert switching assistant to new UI or keep this skip in this PR, whatever you prefer. If we keep the skip, we probably want to follow-up with fixing those right away. At a glance it's just jsdom missing some methods, so shouldn't be too hard, I'm just trying to keep this PR small

Reply:
I'm working on a follow-up PR so we can just skip this here and I'll rebase